netbox/dcim/tables/devices.py

```diff
@@ -390,7 +390,10 @@ class Meta(DeviceComponentTable.Meta):
         )
         default_columns = ('pk', 'name', 'label', 'type', 'speed', 'description', 'cable', 'connection')
         row_attrs = {
-            'class': get_cabletermination_row_class
+            'data-name': lambda record: record.name,
+            'data-mark-connected': lambda record: "true" if record.mark_connected else "false",
+            'data-cable-status': lambda record: record.cable.status if record.cable else "",
+            'data-type': lambda record: record.type
```

Comment:
Lots of static duplication here. Can we replace these with a function that returns the dict for easy reuse?

Reply:
Yeah, I will see what I can do.